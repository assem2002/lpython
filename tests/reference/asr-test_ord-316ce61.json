{
    "basename": "asr-test_ord-316ce61",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_ord.py",
    "infile_hash": "5ea1a821ece44ef24e91e5ba66152fd1c27bb3724d2cad2e7e2d96f2",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_ord-316ce61.stderr",
    "stderr_hash": "46657c4e3818a75f46a02f519ec235038d7b7b026dc486817d2457c8",
    "returncode": 2
}