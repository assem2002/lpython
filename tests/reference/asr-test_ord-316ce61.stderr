semantic error: ord() is only supported for `str` of length 1
 --> tests/errors/test_ord.py:3:13
  |
3 |     i = ord('ab')
  |             ^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
